examples/make_assets/structure_viz.py

```diff
@@ -30,6 +30,9 @@
 
 
 # %% Plot some disordered structures in 2D
+if os.environ.get("MP_API_KEY") is None:
```

Comment:
> not sure that's the root cause here though. MP_API_KEY should be available before python is invoked

100% agree, I don't think that's the issue (though not sure how and when GitHub runner injected env variables). I inserted an `assert mp_api_key is not None` before calling `MPRester` and it seems to pass? I'm super confused at this moment :(